### PR TITLE
Use existing gotchi

### PR DESCRIPTION
**Pull request by cedric-h**

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -627,15 +627,21 @@ impl CountProbability {
...
+    /// This field dictates how much less xp is earned per item in a yield.
+    /// The formula used is xp / (i * dropoff), where xp is an amount in the bounds below,
+    /// and i is the index of the item.
+    pub dropoff: f32,
     /// An upper and lower bound for a random amount of xp to be awarded should this yield occur
-    /// (as determined by the chance field)
-    xp: (usize, usize),
+    /// (as determined by the chance field).
+    /// The xp is awarded on a per item basis, although dropoff can be used to ensure less and
+    /// lesss xp is awarded per item.
```
Comment (Muirrum) [inline src/config.rs]: minor spelling mistake here

File: src/possess/gotchi.rs [hunk 2]
```diff
@@ -165,8 +165,8 @@ fn gotchi_serialize() -> Result<(), AttributeParseError> {
         CONFIG
             .possession_archetypes
             .iter()
-            .position(|x| x.name == "Adorpheus")
-            .unwrap(),
+            .position(|x| x.name == "Gempheus")
```
Comment (Muirrum) [inline src/possess/gotchi.rs:168]: Why was this changed from Adorpheus to Gempheus?

**Review by Muirrum — DISMISSED** [review]
Just some clarifying questions before I approve this

File: src/possess/gotchi.rs (hunk 2 above)
Comment (cedric-h) [reply src/possess/gotchi.rs:168]: Because Adorpheus was removed from the game because we weren't sure how to introduce her to the loot tables.

Comment (cedric-h) [reply src/possess/gotchi.rs:168]: Which is why the test was failing, it couldn't (for very valid reasons) find any gotchi by the name of Adorpheus.

File: src/config.rs (hunk 1 above)
Comment (cedric-h) [reply src/config.rs]: grr
